app/src/pages/Play/YourTurnContent.tsx

```diff
@@ -333,6 +350,12 @@ function YourTurnContent(props: {
             </Grid>
           )}
         </Grid>
+
+        {props.activeTurnPlayState === ActiveTurnPlayState.Playing && (
+          <div style={{ textAlign: "center", color: grey[500] }}>
+            [ Hint: Press the spacebar for "Correct", and S for "Skip" ]
```

Comment:
I think we might want some sort of mobile friendly design here, see examples:

in max reduced width safari (e.g. side by side w/Zoom call)

![image](https://user-images.githubusercontent.com/2534903/80422718-8750a200-8893-11ea-802a-65cc0d34cdf5.png)

in mobile web on Chrome (iPhone X)

![image](https://user-images.githubusercontent.com/2534903/80422911-da2a5980-8893-11ea-9cc7-2f1536f91dad.png)

Couple suggestions to make it more mobile friendly where the real estate is slim and content should be relevant... 

1. (favorite) This message only appears on certain screen sizes (proxy for keyboard is available) or on any screen that's not a mobile device based on user agent (proxy for no keyboard available) something simple like https://stackoverflow.com/questions/11381673/detecting-a-mobile-browser (http://detectmobilebrowsers.com/).

2. We can create a new dismiss-able info card (like the Round info) - that appears if it's your first turn ever

![image](https://user-images.githubusercontent.com/2534903/80423882-9173a000-8895-11ea-8925-5d207302e1eb.png)

3. We hide this in a bottom right gear settings (among other things) - this is the long term direction, where I'd like to put Host controls, and other settings (like adjust turn time, add new player, current score board or stats, etc.).

(Just brainstorming, lmk ur thoughts @wtachau )

Reply:
Ah, great call -- I thought about this when adding it then totally forgot about it before submitting.

I also agree with 1). I don't think keyboard shortcuts are particularly valuable on mobile, since the act of pulling up a keyboard (and how would you do that?) and tapping a letter is more friction than just hitting those big friendly buttons. Plus, it's easier 😄.